container/features/src/main/resources/features.xml

```diff
@@ -1077,4 +1077,11 @@
       <bundle>mvn:org.opennms.features.poller/org.opennms.features.poller.shell/${project.version}</bundle>
     </feature>
 
+    <feature name="opennms-telemetry-jti" description="OpenNMS :: Telemetry :: JTI" version="${project.version}">
```

Comment:
Usually we let the `maven-features-plugin` take care of the feature generation. Is there a particular reason why the feature is defined manually?

Reply:
Personally I like to avoid using our `maven-features-plugin`: it adds unnecessary bundles, and doesn't reference other features that already exist.

Reply:
If it is setup correctly it will not add unnecessary bundles. It only adds transitive dependencies if the scope is not set to provided. Also features are supported. Only thing I see which makes it hard, is that you cannot define a version on a feature.

Reply:
I always had features defined here except for some of the minion features.  Unless there is specific advantage, I will keep it here.

Reply:
The specific advantage is, that the `maven-features-plugin` will add/remove a bundle to the definition every time the pom of the module changes.